Source/WebCore/platform/graphics/texmap/BitmapTexture.h

```diff
@@ -90,12 +98,14 @@ class BitmapTexture final : public ThreadSafeRefCounted<BitmapTexture> {
 
     ClipStack& clipStack() { return m_clipStack; }
 
-    void copyFromExternalTexture(GLuint textureID);
```

Comment:
This change could be moved into a separated PR, to avoid the change of the single call site affected by the removal: `Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedPlatformLayerBufferVideo.cpp`. However, the arguments were wrongly named, and this caused confusion, when trying to add MemoryMappedGPUBuffer awareness to those functions, just avoid those call sites, and make the sizing requirements explicit, you need a _targetRect_ and a _sourceOffset_ to invoke the `copyFromExternalTexture` operation.